docs/api-rule-cr.md

```diff
@@ -0,0 +1,311 @@
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1alpha1
+kind: APIRule
+metadata:
+  name: service-secured
+spec:
+  gateway: kyma-system/kyma-gateway
+  service:
+    name: foo-service
+    port: 8080
+    host: foo.bar
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      mutators: []
+      accessStrategies:
+        - handler: oauth2_introspection
+          config:
+            required_scope: ["read"]
+```
+
+  </details>
+</div>
+
+## Specification
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+| Field   |      Mandatory      |  Description |
+|---|:---:|---|
+| **metadata.name** | **YES** | Specifies the name of the exposed API. |
+| **spec.gateway** | **YES** | Specifies the Istio Gateway. |
+| **spec.host** | **YES** | Specifies the service's communication address for inbound external traffic. If only the leftmost label is provided, the default domain name will be used. |
+| **spec.service.name** | **NO** | Specifies the name of the exposed service. |
+| **spec.service.namespace** | **NO** | Specifies the Namespace of the exposed service. |
+| **spec.service.port** | **NO** | Specifies the communication port of the exposed service. |
+| **spec.rules** | **YES** | Specifies the list of Oathkeeper access rules. |
+| **spec.rules.service** | **NO** | Services definitions at this level have higher precedence than the service definition at the **spec.service** level.|
+| **spec.rules.service.name** | **NO** | Specifies the name of the exposed service. |
+| **spec.rules.service.namespace** | **NO** | Specifies the Namespace of the exposed service. |
+| **spec.rules.service.port** | **NO** | Specifies the communication port of the exposed service. |
+| **spec.rules.path** | **YES** | Specifies the path of the exposed service. |
+| **spec.rules.methods** | **NO** | Specifies the list of HTTP request methods available for **spec.rules.path**. |
+| **spec.rules.mutators** | **NO** | Specifies the list of [Oathkeeper](https://www.ory.sh/docs/next/oathkeeper/pipeline/mutator) or Istio mutators. |
+| **spec.rules.accessStrategies** | **YES** | Specifies the list of access strategies. Supported are [Oathkeeper](https://www.ory.sh/docs/next/oathkeeper/pipeline/authn) `oauth2_introspection`, `jwt`, `noop` and `allow`. We also support `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy. |
+
+>**CAUTION:** If `service` is not defined at **spec.service** level, all defined rules must have `service` defined at **spec.rules.service** level, otherwise the validation fails.
+
+>**CAUTION:** We do not support having both Oathkeeper and Istio `jwt` access strategies at the same time. Access strategies `noop` or `allow` can **not** be used with any other on the same `spec.rules.path`.
+
+### JWT access strategy
+
+>**CAUTION:** Istio JWT is **not** production ready feature and API can change
```

Comment:
please move this note to `Istio JWT configuration`

Reply:
Done.